game/src/Game/GameSession/BillboardCamera.gd

```diff
@@ -0,0 +1,18 @@
+extends Camera3D
```

Comment:
Is `BillboardCamera` actuall used anywhere? I can see there's an instance of it in `GameSession`, but that's not connected/used by anything else from what I can see? Unless it's something to do with the cull mask being `~2` and BillboardManager having layer `2`?

Reply:
To render the billboards on top of everything I found I needed a subviewport. Each subviewport needs its own camera, so I give it a camera, and make it copy the transform of the ordinary camera.
The cull mask is so that only this second camera renders the billboards.

Reply:
As per Spartan's comments the subviewport was unnecessary after all and has been removed along with this camera script.